docs/releases/1.11.txt

```diff
@@ -439,3 +439,7 @@ Miscellaneous
 
 * ``django.test.runner.setup_databases()`` is moved to
   :func:`django.test.utils.setup_databases`. The old location is deprecated.
+
+* Django stopped checking the current user language in
```

Comment:
This reads more like a backwards-incompatible change than a deprecation.
Also, it would be helpful to include an explanation of the reason for the deprecation.